sites/includes/i18n-core.liquid

```diff
@@ -61,7 +63,9 @@
 	{% assign i18nText-signAccount = "My account" %}
 
 	{% assign i18nText-footerSite = "About this site" %}
-	{% assign i18nText-top = "Top of Page" %}
+	{% assign i18nText-themeLinks = "Theme links" %}
+	{% assign i18nText-supportLinks = "Support links" %}
+	{% assign i18nText-transparency = "Transparency" %}
```

Comment:
Those are specific to GCWeb, it should be defined in i18n.liquid file instead of the i18n-core.liquid file.